US881996: Replace gosu with su-exec (#74)
caf-archive/opensuse-base-image: release-notes-2.10.0.md

${version-number}

#### New Features

> - **US881996**: Replace gosu with su-exec  

Reviewer: ```suggestion
- US881996: Replaced `gosu` with `su-exec`  
```

>   - `gosu` utility has been replaced with a much similar and simpler `su-exec`.

Reviewer: This doesn't really make sense, I would just write:
```suggestion
  - The `gosu` utility has been replaced with `su-exec`.
```
